README.md

```diff
@@ -262,3 +262,41 @@ So, this is a full SQL insert for getting only new local posts using a POST requ
 INSERT INTO webhooks (url, method, body_expression, filter_expression, object_type, operation, headers, enhanced_filter)
 VALUES ('https://example.com/webhook', 'POST', 'data.data', 'data.data.local', 'comment', 'INSERT', null, null);
 ```
+
+## RabbitMQ example
+
+You can use RabbitMQ as a queue instead of Redis:
+
+> This is the example from the top of this README modified to include RabbitMQ
+
+```yaml
+services:
+  # ...
+  redis:
```

Comment:
shouldn't redis be excluded here if rmq is used as an alternative to redis?

Reply:
I've added it to the README